WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -243,8 +244,10 @@ + (NSArray *)routes
   // what ios-driver did and sadly, we must copy them.
   NSString *const name = request.arguments[@"name"];
   if (name) {
-    NSPredicate *predicate = [NSPredicate predicateWithFormat:@"%K == %@", FBStringify(XCUIElement, wdName), name];
-    XCUIElement *childElement = [[[[element descendantsMatchingType:XCUIElementTypeAny] matchingPredicate:predicate] allElementsBoundByIndex] lastObject];
+    XCUIElement *childElement = [[[[element descendantsMatchingType:XCUIElementTypeAny] matchingIdentifier:name] allElementsBoundByIndex] lastObject];
```

Comment:
I assume search by identifier (which is the same values as in _wdName_) works faster and also we managed to get rid of one extra code line 💯